src/tools/pmempool/convert.c

```diff
@@ -42,7 +42,11 @@
 #include <getopt.h>
 #include <stdbool.h>
 #include <sys/mman.h>
+#ifdef __FreeBSD__
+#include <sys/endian.h>
```

Comment:
Perhaps this could go into a file at `src/freebsd/endian.h`.
I also made a version that provides some defines using what is available in MacOSX, under a separate directory (not merged yet, might never be).
There is already a variant for Windows.
Perhaps it looks like overkill to have a header, that just includes another header, but if it saves us from all these ifdefs...